content/en/docs/High Availability/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "High Availability"
+linkTitle: "High Availability"
+weight: 170
+description: >
+  Learn how to configure High Availability in Kyverno
+---
+
+## Configure Kyverno in HA mode
+To install Kyverno in HA using Helm Chart.
+  
+  **NOTE:** Due to some cmplexities accompanied with running 2 replicas, the recommended replica counts for HA is at least 3.
+
+``` 
+helm install kyverno kyverno/kyverno --create-namespace --set=replicaCount=3 
+```
+## How HA works in Kyverno
+This section provides details on how Kyverno handles HA scenario.
+- ### Module - Webhook Server
+  Webhook server is where Kyverno receives and processes admission requests. This controller does not require a leader-election. When Kyverno runs multiple instances, the Service will distribute the admission requests across different instances. 
+
+  For “mutate” and “validate” enforce policies, Kyverno returns the decision along with the admission response; it is a synchronous request-response process. However, for “generate” and “validate” audit policies, Kyverno pushes these requests to a queue, and returns the response immediately, and then starts processing the data asynchronously. 
```

Comment:
Here and throughout, it looks like your editor is using smart quotes which tend to cause problems in Markdown engines. See what settings you can tweak to turn those double and single quotes into straight quotes.